documentation/src/components/Sights/TopBar/TopBar.tsx

```diff
@@ -2,13 +2,13 @@ import { labels, sights, vehicles } from '@monkvision/sights';
 import { SearchBar } from '@site/src/components';
 import { SightCard } from '@site/src/components/Sights/SightCard';
 import clsx from 'clsx';
-import React, { ReactElement, useRef } from 'react';
+import React, { FunctionComponentElement, useRef } from 'react';
 import styles from './TopBar.module.css';
 
 export interface ListItem {
   id: string;
   lookups: string[];
-  component: ReactElement;
+  component: FunctionComponentElement<typeof SightCard>;
```

Comment:
Why did you change this exactly? I think `component` could be any type of displayable element. Not ONLY a `SightCard` component

Reply:
Technically it can be any component we wanted. but we are only using SightCard from `pages/Sights/index.tsx` its not clear. We need to go and look into other files to find what are the components we are using. 
Now we can see that in index.tsx file itself we can see the component using

Reply:
I think we just need to think about the objective of the ListItem object. What is its purpose? It is just an item in the list that contains a `component` property that is any React element to be displayed on the screen that's it. So let's just make it like that. I see no advantage of strongly typing the `component` property